### PR TITLE
updated the shared image to show cc logo

### LINKED ISSUE #621: Update the shared image to show CC logo
## Problem
CC sites are not consistent when showing the shared image on social media posts.

## Description
For this first version, we decided to show one single image with the CC logo for all CC sites. Here is attached the JPG file following the [Linkedin recommendations](https://www.linkedin.com/help/linkedin/answer/46687/making-your-website-shareable-on-linkedin?lang=en).

[cclogo-shared-image.jpg.zip](https://github.com/creativecommons/vocabulary/files/5143208/cclogo-shared-image.jpg.zip)

Thanks @zackkrida for bringing this

### PR DESCRIPTION
**Pull request by ooyeraju**
## Fixes

Fixes #621  by @panchovm 

## Description
Updated the shared image to show cc logo

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by zackkrida — APPROVED** [review]

**Review by dhruvkb — APPROVED** [review]

File: packages/vocabulary/.storybook/manager-head.html [hunk 1]
```diff
@@ -6,10 +6,10 @@
 <meta property="og:type" content="website">
 <meta property="og:url" content="https://cc-vocabulary.netlify.com/">
 <meta property="og:description" content="A cohesive design system to unite the web facing Creative Commons">
-<meta property="og:image" content="https://cc-vocabulary.netlify.com/titlecard.png">
+<meta property="og:image" content="https://cc-vocabulary.netlify.com/cclogo-shared-image.jpg">
```
Comment (dhruvkb) [inline packages/vocabulary/.storybook/manager-head.html:9]: This same change would also have to be made in the Storybook for `fonts` and `vue-vocabulary`.

**Review by dhruvkb — CHANGES_REQUESTED** [review]

File: packages/vocabulary/.storybook/manager-head.html (hunk 1 above)
Comment (ooyeraju) [reply packages/vocabulary/.storybook/manager-head.html:9]: Done.

File: packages/vue-vocabulary/.storybook/manager-head.html [hunk 2]
```diff
@@ -6,10 +6,10 @@
...
 <meta property="og:description" content="A Vue component library implementing a cohesive design system to unite the web facing Creative Commons">
-<meta property="og:image" content="https://cc-vue-vocabulary.netlify.com/titlecard.png">
+<meta property="og:image" content="https://cc-vocabulary.netlify.com/cclogo-shared-image.jpg">
 
 <!-- Meta info for Twitter -->
 <meta name="twitter:title" content="vue-vo·cab·u·lar·y">
 <meta name="twitter:card" content="summary_large_image">
 <meta name="twitter:description" content="A Vue component library implementing a cohesive design system to unite the web facing Creative Commons">
-<meta name="twitter:image" content="https://cc-vue-vocabulary.netlify.com/titlecard.png">
+<meta property="og:image" content="https://cc-vocabulary.netlify.com/cclogo-shared-image.jpg">
```
Comment (dhruvkb) [inline packages/vue-vocabulary/.storybook/manager-head.html]:
```suggestion
<meta property="og:image" content="https://cc-vue-vocabulary.netlify.com/cclogo-shared-image.jpg">
```

File: packages/fonts/.storybook/manager-head.html [hunk 3]
```diff
@@ -6,10 +6,10 @@
...
 <meta property="og:description" content="A collection of typefaces for the web facing Creative Commons">
-<meta property="og:image" content="https://cc-fonts.netlify.com/titlecard.png">
+<meta property="og:image" content="https://cc-vocabulary.netlify.com/cclogo-shared-image.jpg">
 
 <!-- Meta info for Twitter -->
 <meta name="twitter:title" content="fonts">
 <meta name="twitter:card" content="summary_large_image">
 <meta name="twitter:description" content="A collection of typefaces for the web facing Creative Commons">
-<meta name="twitter:image" content="https://cc-fonts.netlify.com/titlecard.png">
+<meta property="og:image" content="https://cc-vocabulary.netlify.com/cclogo-shared-image.jpg">
```
Comment (dhruvkb) [inline packages/fonts/.storybook/manager-head.html]:
```suggestion
<meta property="og:image" content="https://cc-fonts.netlify.com/cclogo-shared-image.jpg">
```

**Review by dhruvkb — CHANGES_REQUESTED** [review]
Please also add the same image in `packages/fonts/src/assets/`.

File: packages/fonts/.storybook/manager-head.html (hunk 3 above)
Comment (ooyeraju) [reply packages/fonts/.storybook/manager-head.html]: sorry, I didn't notice. :facepalm:  I have made the changes. Thanks for your patience.

**Review by dhruvkb — APPROVED** [review]
👍 LGTM!

**Pull request merged**